stablehlo/dialect/StablehloOps.cpp

```diff
@@ -4834,37 +4913,26 @@ char nonSpatialDimToString(NonSpatialDim dim) {
 void printConvolutionDimensions(AsmPrinter& p, ConvDimensionNumbersAttr dnums) {
   // TODO(b/202040055): we should check the attribute invariant and print the
   // "raw" form if they are violated, otherwise we'll crash here.
-  constexpr int64_t kUnknownDim = std::numeric_limits<int64_t>::min();
   auto printDim =
-      [&](ArrayRef<int64_t> spatialDims,
-          ArrayRef<std::pair<int64_t, NonSpatialDim>> nonSpatialDims) {
-        int64_t numDims = 0;
-        if (!spatialDims.empty()) {
-          numDims =
-              *std::max_element(spatialDims.begin(), spatialDims.end()) + 1;
-        }
-        for (const auto& dim : nonSpatialDims) {
-          numDims = std::max(numDims, dim.first + 1);
-        }
-
-        llvm::SmallVector<int64_t> dims(numDims, kUnknownDim);
+      [&p](ArrayRef<int64_t> spatial_dims,
+           ArrayRef<std::pair<int64_t, NonSpatialDim>> non_spatial_dims) {
```

Comment:
naming convention of spatial_dims and non_spatial_dims. Shouldn't they be camel Case.

Reply:
Fixed. thanks!